src/translations/screens/subscreens/LocationSearch.ts

```diff
@@ -29,11 +29,24 @@ const LocationSearchTexts = {
           `Journey from ${fromName} to ${toName}.`,
         ),
       a11yHint: _(
-        'Aktivér for å søke etter reiser for dette resultatet.',
+        'Aktiver for å søke etter reiser for disse stedene.',
         'Activate to search for trips for these locations.',
       ),
     },
   },
+  locationResults: {
+    a11y: {
+      activateToUse: _(
+        'Aktiver for å bruke dette stedet',
+        'Activate to use this location',
+      ),
+      activateToSearch: (locationName: string) =>
+        _(
+          `Aktiver for å legge ${locationName} i søkefeltet`,
```

Comment:
Aktivér brukes i hovedsak for accessibility hint. Siden dette er på en label ville jeg utelatt det og heller hatt "Legg ${locationName} i søkefeltet".

Reply:
jepp, enig